### PR TITLE
fix[OA-audit-L01]: Limit ability for Full Policy Escalation Manager to setArbitrationResolution

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

OZ identified the following issue:
```
The setArbitrationResolution function in the FullPolicyEscalationManager
contract can be called multiple times for the same requestId . This would allow the contract
owner to change the price for a given requestId , invalidating the resolvedPrice result in
the OptimisticAsserter that was used when the settleAssertion function was
called.
Consider restricting the setArbitrationResolution function to only allow setting the
resolution once per requestId .
```

This PR addresses this issue by implementing the recomended change by only allowing for the owner to set the resolution value once.

### CODE REVIEW

**Review by mrice32 — APPROVED** [review]
LGTM!

File: packages/core/contracts/optimistic-asserter/implementation/escalation-manager/FullPolicyEscalationManager.sol [hunk 1]
```diff
@@ -153,6 +153,7 @@ contract FullPolicyEscalationManager is BaseEscalationManager, Ownable {
         bool arbitrationResolution
     ) public onlyOwner {
         bytes32 requestId = keccak256(abi.encode(identifier, time, ancillaryData));
+        require(arbitrationResolutions[requestId].valueSet == false, "Arbitration resolution already set");
```
Comment (Reinis-FRP) [inline packages/core/contracts/optimistic-asserter/implementation/escalation-manager/FullPolicyEscalationManager.sol:156]: Shall we better use shorter revert message to fit in 32 bytes, e.g. `Arbitration already resolved` or `Resolution already set`

Comment (chrismaree) [reply packages/core/contracts/optimistic-asserter/implementation/escalation-manager/FullPolicyEscalationManager.sol]: good call out. have updated accordingly.

File: packages/core/test/foundry/optimistic-asserter/escalation-manager/FullPolicyEscalationManager.t.sol [hunk 2]
```diff
@@ -18,13 +18,16 @@ contract FullPolicyEscalationManagerTest is CommonOptimisticAsserterTest {
         bytes memory ancillaryData = "ancillary";
 
         vm.expectRevert("Arbitration resolution not set");
-        escalationManager.getPrice(identifier, time, ancillaryData);
 
+        escalationManager.getPrice(identifier, time, ancillaryData);
         escalationManager.setArbitrationResolution(identifier, time, ancillaryData, true);
         assertTrue(escalationManager.getPrice(identifier, time, ancillaryData) == 1e18);
 
+        vm.expectRevert("Arbitration resolution already set"); // Can not re-set the resolution price once set.
```
Comment (Reinis-FRP) [inline packages/core/test/foundry/optimistic-asserter/escalation-manager/FullPolicyEscalationManager.t.sol:26]:
```suggestion
        vm.expectRevert("Arbitration already resolved"); // Can not re-set the resolution price once set.
```

**Review by Reinis-FRP — APPROVED** [review]

**Pull request merged**